test-runner-frontend/src/Views/DashboardUtils.js

```diff
@@ -0,0 +1,185 @@
+const HIGHER_IS_BETTER = 'higherIsBetter';
+const LOWER_IS_BETTER = 'lowerIsBetter';
+
+const modeData = {
+  thrpt: HIGHER_IS_BETTER,
+  avgt: LOWER_IS_BETTER,
+  sample: LOWER_IS_BETTER,
+  ss: LOWER_IS_BETTER,
+  all: LOWER_IS_BETTER,
+};
+
+export const getMachineNames = (testRunResults) => {
+  var machines = [];
+  testRunResults?.forEach((testRunResult) => {
+    Object.keys(testRunResult?.status).forEach((machine) => {
+      if (!machines.includes(machine)) {
+        machines.push(machine);
+      }
+    });
+  });
+
+  return machines;
+};
+
+export const helper = (modeData, isHigherBetter) => {
```

Comment:
can we please rename this function? helper doesnt explain what it is doing.

Reply:
Done.